cli/bin/setup

```diff
@@ -7,15 +7,4 @@ if [ $? -gt 0 ]; then
   exit 1
 fi
 yarn install
-(cd ../exo-add && bin/setup && echo 'exo-add √')
-(cd ../exo-clone && bin/setup && echo 'exo-clone √')
-(cd ../exo-create && bin/setup && echo 'exo-create √')
-(cd ../exo-deploy && bin/setup && echo 'exo-deploy √')
-(cd ../exo-lint && bin/setup && echo 'exo-lint √')
-(cd ../exo-run && bin/setup && echo 'exo-run √')
-(cd ../exo-setup && bin/setup && echo 'exo-setup √')
-(cd ../exo-sync && bin/setup && echo 'exo-sync √')
-(cd ../exo-test && bin/setup && echo 'exo-test √')
-(cd ../exosphere-shared && bin/setup && echo 'exosphere-shared √')
-(node_modules/.bin/build)
-(./bin/build)
+node_modules/.bin/build
```

Comment:
Exactly what @hugobho said. On Travis, if we only made changes to the `cli` repo, we still need to setup all of the other repos too (to get their `dist` folders available) as they are required by `cli` and needed to run tests like the `tutorial.feature`.